src/monospace/index.ts

```diff
@@ -0,0 +1,114 @@
+import fetch from "node-fetch";
+
+import { FirebaseError } from "../error";
+
+import type { GetInitFirebaseResponse, InitFirebaseResponse } from "./interfaces";
+
+const MONOSPACE_DAEMON_PORT = process.env.MONOSPACE_DAEMON_PORT;
```

Comment:
I won't have this variable set when the extension loads, the extension will need to grab it in a different way, then set the env variable, and then call CLI methods. I think the assignment here will be too early since it's in the global space and will happen when the code loads, before I have time to run anything. Can the assignment be done inside `initFirebase()`?

Reply:
@hsubox76 I just changed it